src/Routes/index.js

```diff
@@ -23,7 +24,8 @@ const Routes = () => (
       <Route path="/" render={() => <Header />} />
     </Switch>
     <PrivateRoute path="/profile" component={ProfileView} />
-    <PrivateRoute path="/study/:kfId/" component={NavBarView} />
+    <PrivateRoute path="/study/:kfId(SD_\w{8})/" component={NavBarView} />
+    <PrivateRoute exact path="/study/new-study" component={NewStudyView} />
```

Comment:
Maybe this should be an admin route? https://github.com/kids-first/kf-ui-data-tracker/blob/ace8fc90d2926fefdf91a27801be896895e2a9b1/src/Routes/AdminRoute.js#L27-L34